[perl] configure: error: no, unable to link

I'm trying to build a new package and ran into this issue on armv7l.  Compiling the same new package on x86_64 works fine.
```
checking for perl5... perl
checking for perl suitability... configure: error: no, unable to link
/usr/local/lib/crew/lib/package.rb:258:in `system': Command failed with exit 1: bash
rxvt_unicode failed to build: `CFLAGS="-O2 -pipe -flto -ffat-lto-objects -fPIC -fuse-ld=gold " CXXFLAGS="-O2 -pipe -flto -ffat-lto-objects -fPIC -fuse-ld=gold " FCFLAGS="-O2 -pipe -flto -ffat-lto-objects -fPIC -fuse-ld=gold " FFLAGS="-O2 -pipe -flto -ffat-lto-objects -fPIC -fuse-ld=gold " LDFLAGS="-flto " bash -c ./configure --prefix=/usr/local --libdir=/usr/local/lib --mandir=/usr/local/share/man --build=armv7l-cros-linux-gnueabihf --host=armv7l-cros-linux-gnueabihf --target=armv7l-cros-linux-gnueabihf --program-prefix='' --program-suffix=''` exited with 1                                                                                                                                                                                     
```
New packages below:
```
require 'package'

class Rxvt_unicode < Package
  description 'A fork of the well known terminal emulator rxvt.'
  homepage 'http://software.schmorp.de/pkg/rxvt-unicode.html'
  version '9.30'
  license 'GPL-3'
  compatibility 'all'
  source_url 'http://dist.schmorp.de/rxvt-unicode/rxvt-unicode-9.30.tar.bz2'
  source_sha256 'fe1c93d12f385876457a989fc3ae05c0915d2692efc59289d0f70fabe5b44d2d'

  binary_url ({
  })
  binary_sha256 ({
  })

  depends_on 'gdk_pixbuf'
  depends_on 'libptytty'
  depends_on 'sommelier'

  def self.build
    system "./configure #{CREW_OPTIONS}"
    system 'make'
  end

  def self.check
    system 'make', 'check'
  end

  def self.install
    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
  end
end
require 'package'

class Libptytty < Package
  description 'OS independent and secure pty/tty and utmp/wtmp/lastlog handling.'
  homepage 'http://software.schmorp.de/pkg/rxvt-unicode.html'
  version '2.0'
  license 'GPL-2'
  compatibility 'all'
  source_url 'http://dist.schmorp.de/libptytty/libptytty-2.0.tar.gz'
  source_sha256 '8033ed3aadf28759660d4f11f2d7b030acf2a6890cb0f7926fb0cfa6739d31f7'

  binary_url({
  })
  binary_sha256({
  })

  def self.build
    Dir.mkdir 'builddir'
    Dir.chdir 'builddir' do
      system "cmake #{CREW_CMAKE_OPTIONS} ../ -G Ninja"
    end
    system 'ninja -C builddir'
  end

  def self.install
    system "DESTDIR=#{CREW_DEST_DIR} ninja -C builddir install"
  end
end
```

--- a/packages/libptytty.rb
+++ b/packages/libptytty.rb
@@ -1,0 +1,36 @@
+require 'package'
+
+class Libptytty < Package
+  description 'OS independent and secure pty/tty and utmp/wtmp/lastlog handling.'
+  homepage 'http://software.schmorp.de/pkg/rxvt-unicode.html'
+  version '2.0'
+  license 'GPL-2'
+  compatibility 'all'
+  source_url 'http://dist.schmorp.de/libptytty/libptytty-2.0.tar.gz'
+  source_sha256 '8033ed3aadf28759660d4f11f2d7b030acf2a6890cb0f7926fb0cfa6739d31f7'
+
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libptytty/2.0_armv7l/libptytty-2.0-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libptytty/2.0_armv7l/libptytty-2.0-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libptytty/2.0_i686/libptytty-2.0-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libptytty/2.0_x86_64/libptytty-2.0-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    aarch64: 'd3ec0975d61905992380ef3ffa8820ee680aac02234822aece6f59a00d24d54b',
+     armv7l: 'd3ec0975d61905992380ef3ffa8820ee680aac02234822aece6f59a00d24d54b',
+       i686: 'c211dafd960e7257990f5753424026ebe35ce26ad5164b3b3f213c73e63949e7',
+     x86_64: '80120d3e5fd47dda88681dc20e5b2d92f2b88e93beff5e6777383f802fdb0847'
+  })
+
+  def self.build
+    Dir.mkdir 'builddir'
+    Dir.chdir 'builddir' do
+      system "cmake #{CREW_CMAKE_OPTIONS} ../ -G Ninja"
+    end
+    system 'ninja -C builddir'
+  end
+
+  def self.install
+    system "DESTDIR=#{CREW_DEST_DIR} ninja -C builddir install"
+  end
+end

--- a/packages/perl.rb
+++ b/packages/perl.rb
@@ -4,26 +4,27 @@ class Perl < Package
   description 'Perl 5 is a highly capable, feature-rich programming language with over 29 years of development.'
   homepage 'https://www.perl.org/'
   @_ver = '5.34.1'
-  version @_ver
+  version "#{@_ver}-1"
   license 'GPL-1+ or Artistic'
   compatibility 'all'
   source_url "https://www.cpan.org/src/5.0/perl-#{@_ver}.tar.xz"
   source_sha256 '6d52cf833ff1af27bb5e986870a2c30cec73c044b41e3458cd991f94374039f7'
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/perl/5.34.1_armv7l/perl-5.34.1-chromeos-armv7l.tar.zst',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/perl/5.34.1_armv7l/perl-5.34.1-chromeos-armv7l.tar.zst',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/perl/5.34.1_i686/perl-5.34.1-chromeos-i686.tar.zst',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/perl/5.34.1_x86_64/perl-5.34.1-chromeos-x86_64.tar.zst'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/perl/5.34.1-1_armv7l/perl-5.34.1-1-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/perl/5.34.1-1_armv7l/perl-5.34.1-1-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/perl/5.34.1-1_i686/perl-5.34.1-1-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/perl/5.34.1-1_x86_64/perl-5.34.1-1-chromeos-x86_64.tar.zst'
   })
   binary_sha256({
-    aarch64: 'dfffdd358c4863f28a7865e7087f6a716478509b7ebeafab0b1200bc5e65fe3d',
-     armv7l: 'dfffdd358c4863f28a7865e7087f6a716478509b7ebeafab0b1200bc5e65fe3d',
-       i686: '49c0701b82abe192ec20a22f435df153689c7269f36da9ab38299496a6360233',
-     x86_64: '09abf3ae359466bb0451e9ed890b4d95bc7721de05fa65f8acba5ee131797b46'
+    aarch64: '6c8b5d157e1e2fd3ebd3ee55a8ef78a516339ddd55a4f6ebef2f1e4b5d9812a9',
+     armv7l: '6c8b5d157e1e2fd3ebd3ee55a8ef78a516339ddd55a4f6ebef2f1e4b5d9812a9',
+       i686: '71f484904c833ac564961903ee1d05b2f8a7d0df8358b9dde8b94bfd2df47cac',
+     x86_64: '4c0bc04df5a046bffe2e35b5fc6bfec8cfbe462c5cc4e9b2fb9e7e70c8373828'
   })
 
   depends_on 'patch' => :build
+  no_patchelf
 
   def self.build
     FileUtils.ln_sf "#{CREW_LIB_PREFIX}/libnsl.so.1", "#{CREW_LIB_PREFIX}/libnsl.so"
@@ -52,9 +53,12 @@ class Perl < Package
   end
 
   def self.install
+    @perl_fullversion = @_ver.split('-')[0]
+    @perl_version = @_ver.rpartition('.')[0]
+
     system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
     # Make libperl symlinks into standard locations
-    libperl_so = "#{CREW_PREFIX}/lib/perl5/#{@perl_fullversion}/#{ARCH}-linux-thread-multi/CORE/libperl.so"
+    libperl_so = "#{CREW_LIB_PREFIX}/perl5/#{@perl_fullversion}/core_perl/CORE/libperl.so"
     FileUtils.ln_sf libperl_so, "#{CREW_DEST_LIB_PREFIX}/libperl.so.#{@_ver}" # e.g., libperl.so.5.34.1
     FileUtils.ln_sf libperl_so, "#{CREW_DEST_LIB_PREFIX}/libperl.so.#{@_ver.sub(/\.\d$/, '')}" # e.g., libperl.so.5.34
     FileUtils.ln_sf libperl_so, "#{CREW_DEST_LIB_PREFIX}/libperl.so#{@_ver.sub(/\.\d\.\d$/, '')}" # e.g., libperl.so.5

--- a/packages/rxvt_unicode.rb
+++ b/packages/rxvt_unicode.rb
@@ -1,0 +1,41 @@
+require 'package'
+
+class Rxvt_unicode < Package
+  description 'A fork of the well known terminal emulator rxvt.'
+  homepage 'http://software.schmorp.de/pkg/rxvt-unicode.html'
+  version '9.30'
+  license 'GPL-3'
+  compatibility 'all'
+  source_url 'http://dist.schmorp.de/rxvt-unicode/rxvt-unicode-9.30.tar.bz2'
+  source_sha256 'fe1c93d12f385876457a989fc3ae05c0915d2692efc59289d0f70fabe5b44d2d'
+
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/rxvt_unicode/9.30_armv7l/rxvt_unicode-9.30-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/rxvt_unicode/9.30_armv7l/rxvt_unicode-9.30-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/rxvt_unicode/9.30_i686/rxvt_unicode-9.30-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/rxvt_unicode/9.30_x86_64/rxvt_unicode-9.30-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    aarch64: 'b23e4d4e20911d6e8085e8867358f9cb189d599c1733edc994504de55147261a',
+     armv7l: 'b23e4d4e20911d6e8085e8867358f9cb189d599c1733edc994504de55147261a',
+       i686: 'd0d24b033752ea7191b8d2ef8990a79ac14192deb72e231f8cf4e43f1f5530d7',
+     x86_64: '4393a4bdb819213b05deb41b0a713e81b52ad0f786c095dd77fe2f03e0a8452c'
+  })
+
+  depends_on 'gdk_pixbuf'
+  depends_on 'libptytty'
+  depends_on 'sommelier'
+
+  def self.build
+    system "./configure #{CREW_OPTIONS}"
+    system 'make'
+  end
+
+  def self.check
+    system 'make', 'check'
+  end
+
+  def self.install
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+  end
+end

